chart-infra/templates/scale-to-zero.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if (eq .Values.kubernetes.env "staging") }}
+{{- if or (eq .Values.kubernetes.env "staging") (eq .Values.myAccount.accountId "160782110667") }}
```

Comment:
enabling cron for production in HMS

Reply:
Can you explain what does this CRON do?

Reply:
It is because I wonder if we also want it in the community instance